tests/integration/conftest.py

```diff
@@ -148,7 +150,7 @@ async def _wait_for_ingress(ingress_name):
                 for ingress in kube.list_namespaced_ingress(ops_test.model.name).items
             ],
             wait_period=5,
-            timeout=300,
+            timeout=10 * 60,
```

Comment:
This was switched to `10 * 60` whereas other changes it was set to `300` rather than something like `5 * 60`? The `X * 60` to me is quite nice because you can easily tell how many minutes it is without maths 😉

Reply:
I agree. x*60 is easier to ready. Changed them all to be like that and with the same value.